omod-1.10/src/main/java/org/openmrs/module/attachments/rest/AttachmentResource1_10.java

```diff
@@ -250,4 +263,99 @@ protected PageableResult doSearch(RequestContext context) {
+			throw new IllegalRequestException(
+					"The following obs is not a complex obs, no complex data can be retrieved. " + "Obs UUID: "
+							+ obs.getUuid());
+		}
+
+		ComplexViewHelper viewHelper = attachmentsContext.getComplexViewHelper();
+		String view = new String();
+		Obs complexObs = Context.getObsService().getComplexObs(obs.getObsId(), viewHelper.getView(obs, view));
+		ComplexData complexData = complexObs.getComplexData();
+
+		// setup attachment instance
+		attachment = new Attachment();
+		attachment.setComplexData(complexData);
+
+		// Switching to our complex data object
+		ValueComplex valueComplex = new ValueComplex(obs.getValueComplex());
+		AttachmentComplexData attComplexData = attachmentsContext.getComplexDataHelper().build(
+				valueComplex.getInstructions(),
+				complexData);
+
+		// generate the bytes array from the AttachmentComplexData
+		byte[] complexByte = attComplexData.asByteArray();
+
+		// fetch the mime type of the bytes array
+		InputStream is = new BufferedInputStream(new ByteArrayInputStream(complexByte));
+		try {
+			URLConnection.guessContentTypeFromStream(is);
+		} catch (IOException e) {
+			e.printStackTrace();
+		}
+
+	}
+
+	@PropertyGetter("bytesMimeType")
+	public static String getBytesMimeType(Attachment attachment) {
+		return attachment.getBytesMimeType();
+	}
+
+	/**
+	 * Sets the bytes contentFamily of an attachment
+	 * 
+	 * @param the           attachment instance
+	 * @param contentFamily of the bytes to set
+	 */
+	@PropertySetter("bytesContentFamily")
+	public void setBytesContentFamily(Attachment attachment, ContentFamily bytesContentFamily) {
+		Obs obs = new Obs();
+		obs = attachmentsContext.getObsService().getObsByUuid(obs.getUuid());
+
+		if (!obs.isComplex()) {
+			throw new IllegalRequestException(
+					"The following obs is not a complex obs, no complex data can be retrieved. " + "Obs UUID: "
+							+ obs.getUuid());
+		}
+
+		ComplexViewHelper viewHelper = attachmentsContext.getComplexViewHelper();
+		String view = new String();
+		Obs complexObs = Context.getObsService().getComplexObs(obs.getObsId(), viewHelper.getView(obs, view));
+		ComplexData complexData = complexObs.getComplexData();
+
+		// setup attachment instance
+		attachment = new Attachment();
+		attachment.setComplexData(complexData);
+
+		// Switching to our complex data object
+		ValueComplex valueComplex = new ValueComplex(obs.getValueComplex());
+		AttachmentComplexData attComplexData = attachmentsContext.getComplexDataHelper().build(
+				valueComplex.getInstructions(),
+				complexData);
+		byte[] complexByte = attComplexData.asByteArray();
+
+		// convert the byte array into a string
+		String str = new String(complexByte, StandardCharsets.UTF_8);
+
+		// fetch the content family
+		attachmentsContext.getContentFamily(str);
+	}
+
+	@PropertyGetter("bytesContentFamily")
+	public static ContentFamily getBytesContentFamily(Attachment attachment) {
+		return attachment.getBytesContentFamily();
+	}
```

Comment:
Again, we won't need all this. Specifically you can't set the MIME type of an attachment, it is inferred from the file in the first place. This already happens, nothing to do as part of this PR.